lib/content._coffee

```diff
@@ -47,11 +44,14 @@ getFilePathForId = (id) ->
     id = id.replace /([A-Z])/g, '_$1'
     Path.join DIR_BY_ID_PATH, "#{id}.json"
 
-getFilePathForURL = (url) ->
-    if idFile = DATABASE["#{hashURL url}.json"]
-        Path.join DIR_BY_ID_PATH, idFile.replace '../content-by-id', ''
-    else
-        ''
+readFileForURL = (url, _) ->
+    cloudClient.download
+        container: 'content/content-by-url'
+        remote: "#{hashURL url}.txt"
+
+    # Somehow after the download we get the ID and then ...
+    path = Path.join DIR_BY_ID_PATH, "#{id}.json"
+    readFile path, _
```

Comment:
So bypassing the _ into here the actual data we want magically appears where it's needed by whoever called readFileForURL, correct?